app/schemas/models/standards_correlation.schema.js

```diff
@@ -0,0 +1,90 @@
+const c = require('./../schemas')
+
+const StandardsCorrelation = c.object(
+  {
+    description: '',
+    title: 'Standards Correlation',
+    required: ['name']
+  },
+  {
+    subject: c.shortString({
+      title: 'Subject',
+      type: 'string',
+      enum: ['computer-science', 'literacy', 'math', 'science']
+    }),
+    district: c.objectId({
+      links: [{ rel: 'extra', href: '/db/district/{($)}' }],
+      title: 'District ID',
+      description: 'The ID of the district that this correlation belongs to'
+    }),
+    administrativeRegion: c.shortString({
+      title: 'Administrative Region',
```

Comment:
We can probably say "State" here for the title and put "Administrative Region (state, province, prefecture, etc.)" in the description, since we're largely going to use these for US states.

Reply:
Also, we should have some way to make sure that we are using the codes (`"TX"` instead of `"Texas"`), otherwise data are going to get muddled. We may have solved this elsewhere with dropdowns?